lib/ansible/modules/cloud/cloudstack/cs_host.py

```diff
@@ -29,63 +15,73 @@
 short_description: Manages hosts on Apache CloudStack based clouds.
 description:
   - Create, update and remove hosts.
-version_added: "2.3"
-author: "René Moser (@resmo)"
+version_added: '2.3'
+author: René Moser (@resmo)
 options:
   name:
     description:
       - Name of the host.
+    type: str
     required: true
-    aliases: [ 'ip_address' ]
+    aliases: [ ip_address ]
   url:
     description:
       - Url of the host used to create a host.
-      - If not provided, C(http://) and param C(name) is used as url.
-      - Only considered if C(state=present) and host does not yet exist.
+      - If not provided, C(http://) and param I(name) is used as url.
+      - Only considered if I(state=present) and host does not yet exist.
+    type: str
   username:
     description:
       - Username for the host.
-      - Required if C(state=present) and host does not yet exist.
+      - Required if I(state=present) and host does not yet exist.
+    type: str
   password:
     description:
       - Password for the host.
-      - Required if C(state=present) and host does not yet exist.
+      - Required if I(state=present) and host does not yet exist.
+    type: str
   pod:
     description:
       - Name of the pod.
-      - Required if C(state=present) and host does not yet exist.
+      - Required if I(state=present) and host does not yet exist.
+    type: str
   cluster:
     description:
       - Name of the cluster.
+    type: str
   hypervisor:
     description:
       - Name of the cluster.
-      - Required if C(state=present) and host does not yet exist.
-    choices: [ 'KVM', 'VMware', 'BareMetal', 'XenServer', 'LXC', 'HyperV', 'UCS', 'OVM', 'Simulator' ]
+      - Required if I(state=present) and host does not yet exist.
+    type: str
+    choices: [ KVM, VMware, BareMetal, XenServer, LXC, HyperV, UCS, OVM, Simulator ]
   allocation_state:
     description:
       - Allocation state of the host.
-    choices: [ 'enabled', 'disabled' ]
+    type: str
+    choices: [ enabled, disabled, maintenance ]
   host_tags:
     description:
       - Tags of the host.
+    type: list
     aliases: [ host_tag ]
   state:
     description:
       - State of the host.
-    default: 'present'
-    choices: [ 'present', 'absent' ]
+    type: str
+    default: present
+    choices: [ present, absent ]
   zone:
     description:
       - Name of the zone in which the host should be deployed.
       - If not set, default zone is used.
+    type: str
 extends_documentation_fragment: cloudstack
 '''
 
 EXAMPLES = '''
 - name: Ensure a host is present but disabled
-  local_action:
-    module: cs_host
+  cs_host:
     name: ix-pod01-esx01.example.com
     cluster: vcenter.example.com/ch-zrh-ix/pod01-cluster01
```

Comment:
These names look a bit too familiar to me, I think they are based on actual host names...
If you're cleaning up the documentation, may I suggest anonymising them a bit more?
Something like pod01.zone01.example.com and hv.example.com/zone01/cluster01

Reply:
Done!

Reply:
I'll pay attention to that for the next modules.

Reply:
example.com?

Reply:
There's actually an RFC describing reserved example DNS names: https://tools.ietf.org/html/rfc2606
example.com is widely used in documentation for exactly this purpose.